### DISCUSSION TITLE
JConf Dominicana 2024 Important Dates

### DISCUSSION THREAD

**Discussion by ecabrerar** [JavaDominicano/java-dominicano-comite-ejecutivo#26] [Ideas]
**Option 1:**

JConf Dominicana 2024
19-20, July 2024

CFP Opens: December 27, 2023 00:00 AST
CFP Closes: February 27, 2024 11:02 AST

Speakers and Talks Selection: March 1, 2024
Initial Schedule Posting: March 15,2024

Final Schedule Posting: June 19, 2024

Last change registration ends: July 12, 2024

**Option 2:**

JConf Dominicana 2024
19-20, July 2024

CFP Opens: December 19, 2023 00:00 AST
CFP Closes: February 19, 2024 11:02 AST

Speakers and Talks Selection: February 23, 2024
Initial Schedule Posting: March 01,2024

Final Schedule Posting: June 19, 2024

Last change registration ends: July 12, 2024

**Comment by ecabrerar** [comment]
Estas son las dos opciones, la primera opción, es tomando en cuenta la fecha de la primera versión de jconf dominicana (2019).
Favor dejar sus opiniones

**Comment by fredpena** [comment]
Me gustan las dos opciones, quizá el CFP cierre muy pronto, es mi única observación. Pero después todo lo veo bien

**Comment by ecabrerar** [comment]
Viendo que en diciembre y parte de enero, los locales no están en eso, nos conviene que cierre un poco tarde en febrero

**Comment by fredpena** [comment]
A lo que me refiero es que debemos analizar si valdría la pena cerrar en Marzo, no en febrero. Por eso es que digo que siento que cierra muy pronto.

**Reply by BrayanMnz** [reply]
El tema de cerrar pronto, es que quien viene de fuera tenga la anticipación adecuada para tomar su decisión.

**Comment by ecabrerar** [comment]
Es que para los locales, podemos enviarle algunos enlaces especiales

**Reply by fredpena** [reply]
Me gusta el plan, estoy de acuerdo con eso que plantea.

**Comment by ecabrerar** [comment]
Siempre habrán personas que se van a quedar y van a escribir después de cerrar.
